KsApi/models/Location.swift

```diff
@@ -31,3 +31,9 @@ extension Location: Equatable {}
 public func == (lhs: Location, rhs: Location) -> Bool {
   return lhs.id == rhs.id
 }
+
+extension Location: GraphIDBridging {
+  public static var modelName: String {
+    return "Location"
```

Comment:
Need to confirm this name.

Reply:
I think the `locationId` mutation input is actually not a `graphID`. In other words you should just be able to use `location.id`.